cfme/tests/intelligence/reports/test_crud.py

```diff
@@ -165,3 +166,43 @@ def test_dashboard_crud():
     with update(d):
         d.widgets = "Top Storage Consumers"
     d.delete()
+
+
+@pytest.mark.uncollectif(lambda: version.current_version() < '5.5')
+def test_run_report(rest_api):
+    report = rest_api.collections.reports.get(name='VM Disk Usage')
+    response = report.action.run()
+
+    @pytest.wait_for(timeout="5m", delay=5, message="REST running report finishes")
+    def _finished():
```

Comment:
Actually, when you are changing this, you can write it like this:

```python
@pytest.wait_for(timeout="5m", delay=5)
def rest_running_report_finishes():
    pass  # Do stuff™
```

It will show up in the logs as `rest_running_report_finishes()` which does not have spaces, but it makes you more meaningful function names.